src/Matterhorn/Types/Posts.hs

```diff
@@ -223,7 +226,13 @@ getAttachmentText p =
     Just attachments ->
       Blocks $ fmap (Blockquote . render) attachments
   where render att = parseMarkdown Nothing (att^.ppaTextL) <>
-                     parseMarkdown Nothing (att^.ppaFallbackL)
+                     parseMarkdown Nothing (att^.ppaFallbackL) <>
+                     parseMarkdown Nothing (toText $ unlines $ toList $ fmap renderAttField (att^.ppaFieldsL))
+
+renderAttField :: PostPropAttachmentField -> String
+renderAttField f = "  * " ++
+  (if (ppafTitle f) == T.empty then "" else printf "**%s:** " (ppafTitle f)) ++
+  printf "%s" (ppafValue f)
```

Comment:
> If you'd rather I do that refactoring, I am happy to do it.

Would you? This is literally the first Haskell code I've ever written, so if you want to have at it, by all means, go ahead. :smile:

Reply:
Okay, I don't mind - but I wanted to be sensitive to whatever you wanted to do in case you had a preference for tackling as a way to get more learning experience. If you did, I wouldn't be as concerned with you getting it exactly right; I would just be happy to provide pointers.

Reply:
Given that, let me know what you would prefer.

Reply:
Hi, sorry for the late reply, it was late at night already but I didn't want to leave you hanging for too long.

I can give it a shot, I might ask for some guidance if I get stuck at something, but I'll try. Thanks for your help so far!

Reply:
I think I'm starting to get the hang of it, it wasn't so bad after all. :smile: